### PR TITLE
Voice updates

### PR DESCRIPTION
**Pull request by quarterback**
Content changes to reflect tone and voice guidelines, mostly.

### CODE REVIEW

File: web/src/i18n/locales/en.json [hunk 1]
```diff
@@ -114,22 +114,22 @@
     "goals": {
       "title": "Needs and Objectives",
       "subheader":
-        "List the goals and objectives you're hoping to accomplish as part of this activity.",
+        "List the goals and objectives you intend to accomplish as part of this activity.",
       "helpText":
```
Comment (NAretakis) [inline web/src/i18n/locales/en.json]: propose this as "subheaderhelptext":

File: web/src/i18n/locales/en.json [hunk 2]
```diff
@@ -114,22 +114,22 @@
...
-        "Please include a brief description of each goal, and how they relate to the vision identified in the Program Summary section\n\n Within the Objectives section describe how the state will measure progress toward the goal identified.",
+        "Please include a brief description of each goal relates to the vision identified in the program summary section.\n\n Within the objectives section, describe how the state will measure progress toward the goal identified.",
       "reminder": "DON'T FORGET: Each goal should have a matching objective.",
       "goalHeader": "Goal #{{number}}",
       "objectiveHeader": "Objective (How the you'll achieve the goal)",
       "helpText":
```
Comment (NAretakis) [inline web/src/i18n/locales/en.json]: propose this as "objectiveheaderhelptext":

Comment (quarterback) [reply web/src/i18n/locales/en.json]: Fixed

**Comment by NAretakis** [comment]
Looks ok, but I can't figure out why it is still failing CircleCI. Any ideas?

**Comment by mgwalker** [comment]
The "deploy pr" step is semi-broken right now - the part of the script that has the bot post a message isn't working for some reason.  All the tests pass and the "deploy pr" step is not required to merge, so this one is okay from that perspective - but we do need to fix that script.  😬

**Comment by NAretakis** [comment]
Gotcha, makes sense. @quarterback do you want to move this one in as is, or should we try to incorporate some of the changes from my branch into this? A question of one or two PRs?

**Comment by NAretakis** [comment]
Actually, two PRs makes sense to me. I'll approve.

**Pull request merged**